syft/frameworks/torch/tensors/Polynomial.py

```diff
@@ -0,0 +1,77 @@
+class PolynomialTensor:
```

Comment:
polynomial tensor should be a subclass of AbstractTensor. Furthermore we probably should have a constructor which takes an order parameter which specifies the order of the polynomial approximation at definition time (with a sensible default of 4 or 5)